packages/theme-compiler/src/less-plugin-inline-svg/encoders.js

```diff
@@ -9,13 +9,11 @@ const encodeHtmlEntities = (input) => {
 };
 
 const optimizeSvg = (input) => {
-  // replace native RegExp with RE2 to prevent ReDos
-  // https://owasp.org/www-community/attacks/Regular_expression_Denial_of_Service_-_ReDoS
-  return input.replace(new RE2(/\s*\n\s*/g), '');
+  return optimize(input).data;
```

Comment:
I think we should include the `optimize` from svgo in  `wrapAndEncodeSvgToBase64` and reduce this function

Reply:
`optimizeSvg` is being used in another location: check `warpSvg` function.

Reply:
OK, I see.